### DISCUSSION TITLE
What happened to version version 1.3.4?

### DISCUSSION THREAD

**Discussion by leicray** [GMOD/jbrowse-components#2319] [Q&A] [answered]
JBrowse versions **appear** to jump from version 1.3.3 to 1.3.5. My installed version (installed using the CLI) reports that it is 1.3.4 but an upgrade to 1.3.5 appears to do absolutely nothing.

I'm a newby and I'm confused.

**Comment by cmdcolin** [comment]
@leicray sorry about the confusion, basically the 1.3.4 version had an issue during release so we kind of "skipped over" 1.3.4. we could have probably documented this better but hopefully there won't be as much confusion in the future (we have now changed it so versions should all increment in lockstep now, so there should be less confusion in the future)

**Comment by leicray** [comment, marked as answer]
Are you saying that although my installed version reports that it's 1.3.4. it's really 1.3.5? I presume that things will sort themselves out when 1.3.6 becomes available.

**Reply by cmdcolin** [reply]
Yep, that is exactly it! And we are hoping to release 1.3.6 soon really here (with the long awaited gene name searching functionality :)
